schemas/pom.xml

```diff
@@ -51,8 +51,29 @@
     <module>dublin-core</module>
     <module>iso19110</module>
     <module>iso19139</module>
-    <module>iso19115-3.2018</module>
   </modules>
+
+  <profiles>
+    <profile>
+      <id>iso19115-3.2018</id>
```

Comment:
I like the idea that you are using the same id for both web and schema pom but unfortunately it does not work - only one is activated.  So it may need to be renamed - maybe schema-iso19115-3.2018 and module-iso19115-3.2018?

Reply:
I think they both should activate (we use the same pattern in geoserver codebase). I am going to double check if my automatic automation is wrong :P

Reply:
I have successfully used the same profile name in both schemas/pom.xml and web/pom.xml.

I tested by excluding on the command line:

```
mvn clean install -P-schema-iso19115-3-2018
cd web
mvn jetty:run -P-schema-iso19115-3-2018
```